python/lib/sift_py/ingestion/config/yaml/spec.py

```diff
@@ -90,6 +91,29 @@ class FlowYamlSpec(TypedDict):
     channels: List[ChannelConfigYamlSpec]
 
 
+class ReportTemplateYamlSpec(TypedDict):
+    """
+    Formal spec for a report template.
+
+    `name`: Name of the report template.
+    `template_client_key`: Unique client key to identify the report template.
+    `organization_id`: Organization ID that the report template belongs to.
+    `tags`: Tags to associate with the report template.
+    `description`: Description of the report template.
+    `rule_client_keys`: List of rule client keys associated with the report template.
+    `archived_date`: Date when the report template was archived. Setting this field
+      will archive the report template, and unsetting it will unarchive the report template.
+    """
+
+    name: str
+    template_client_key: str
+    organization_id: str
```

Comment:
Organization ID should be marked as `NotRequired`. We should reflect that in the doc comments as well.